modules/local/fusioncatcher/download/main.nf

```diff
@@ -2,17 +2,17 @@ process FUSIONCATCHER_DOWNLOAD {
     tag "fusioncatcher_download"
     label 'process_medium'
 
-    conda "${projectDir}/environment.yml"
+    conda "${moduleDir}/environment.yml"
     container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
-        'https://community-cr-prod.seqera.io/docker/registry/v2/blobs/sha256/d5/d53f36e9e01d14a0ae8e15f8046f52b2883c970c27fe43fdfbd9440a55f5403f/data' :
-        'community.wave.seqera.io/library/fusioncatcher:1.33--4733482b637ef92f' }"
+        'https://depot.galaxyproject.org/singularity/fusioncatcher:1.33--hdfd78af_5':
+        'biocontainers/fusioncatcher:1.33--hdfd78af_5' }"
```

Comment:
Any reason behind this change? Since it is all wget, perhaps we shouldn't even have this, right?

Reply:
The old container didn't have `wget`, but yes you are right it doesn't need the whole dependency of fusioncatcher

Reply:
I updated it, now it only uses wget